pkg/exchange/binance/stream.go

```diff
@@ -304,7 +304,7 @@ func (s *Stream) getEndpointUrl(listenKey string) string {
 
 	if s.IsFutures {
 		url = FuturesWebSocketURL + "/ws"
-	} else if isBinanceUs() {
+	} else if util.IsBinanceUs() {
```

Comment:
isBinanceUs should only be used inside the binance package, only common functions could be moved into the util package

Reply:
ok, moved back, done